Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:20
+
+WORKDIR /app
+
+COPY package*.json ./
+COPY prisma ./prisma/
+COPY . .
+
+# Copy the .env file
+COPY .env .env
+
+# Install dependencies
+RUN npm install
+
+# Run database migrations and seed the database
+RUN npx prisma db push && npx prisma db seed
```

Comment:
nice work

Reply:
@VineeTagarwaL-code the changes are made accordingly, 
1. changes made in the docker file
<img width="1071" alt="Screenshot 2024-08-07 at 7 11 48 AM" src="https://github.com/user-attachments/assets/b07bde57-0e20-4c46-820d-995782f28463">

2. updated readme.md with -e flag (for environment vars)
<img width="1184" alt="Screenshot 2024-08-07 at 7 11 35 AM" src="https://github.com/user-attachments/assets/592f4316-f2b4-4876-9b6d-9cb06980f804">

Reply:
@VineeTagarwaL-code  also do we need to seed the DB or not, in the above updated version the db seeding is not there, if you want to seed the data (i.e. the users and the password), i need to add the seed command in the Dockerfile. please let me know any changes.

Reply:
@VineeTagarwaL-code  #37 code changes are made accordingly, please review
1. optimized docker file with lighter version of Node
2. dont need to copy .env just pass env (var-args)
3. updated readme.md with updated commands

- for any other changes,  Please advise.

https://github.com/user-attachments/assets/90adaac9-4eed-4947-a825-61a9a99b6895